README.md

```diff
@@ -96,8 +96,7 @@ PaStA understands several mailbox exchange formats:
 #### Patch stack mode
 This mode uses patch stack definition files to compare succesive versions of
 patch stacks. Different patch stacks are defined in the patch stack definition.
-An example configuration, as well as script can be found in the resources of
-the PreemptRT project.
+Example configurations, as well as a scripts can be found in the [PaStA-resources project](https://github.com/lfd/PaStA-resources).
```

Comment:
There are no scripts in pasta-resources. Pasta-resources is not a project, it is just a git repository. The patch stack mode is really only in the PreemptRT config. Let us not do this change. Thanks.

Reply:
Yep, @bulwahn, agree. Will close this. Thanks!